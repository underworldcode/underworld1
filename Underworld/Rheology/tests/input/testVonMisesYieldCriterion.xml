<?xml version="1.0"?>
<!DOCTYPE StGermainData SYSTEM "stgermain.dtd">

<StGermainData xmlns="http://www.vpac.org/StGermain/XML_IO_Handler/Jun2003">

	<include>testYieldCriterion.xml</include>

	<struct name="components" mergeType="merge">
		<struct name="context">
			<param name="Type">UnderworldContext</param>
		</struct>
		<struct name="uzawa" mergeType="merge">
			<param name="tolerance" mergeType="replace">1.0e-4</param>
		</struct>
		<struct name="yieldRheology">
			<!-- These parameters mean that the whole of the box should yield at time = 0.75 -->
			<param name="Type">VonMises</param>
         <!-- Material Points needed for hasYielded variable -->
         <param name="MaterialPointsSwarm">materialSwarm</param>
         <param name="StrainRateField">StrainRateField</param>
			<param name="cohesion">3.0222222</param>
		</struct>	
	</struct>
	<param name="maxTimeSteps" mergeType="replace"> 13 </param>

</StGermainData>
