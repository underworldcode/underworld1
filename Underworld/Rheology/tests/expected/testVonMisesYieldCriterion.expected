Material yielded at time       0.7812 (step 12) within:
	x:        0.402 -        2.598
	y:      0.02846 -        1.318
